docs/metal_device_create.md

```diff
@@ -14,10 +14,10 @@ metal device create -p <project_id> (-m <metro> | -f <facility>) -P <plan> -H <h
 
 ```
   # Provisions a c3.small.x86 in the Dallas metro running Ubuntu 20.04:
-  metal device create -p $METAL_PROJECT_ID -P c3.small.x86 -m da -H test-staging-2 -O ubuntu_20_04
+  metal device create -p $METAL_PROJECT_ID -p c3.small.x86 -m da -H test-staging-2 -O ubuntu_20_04,
```

Comment:
What's up with the commat at the end?

Reply:
[dce7f70](https://github.com/equinix/metal-cli/pull/208/commits/dce7f70e3c0c726cd73a6c5d3214e60b8ea84691) was a fix for #204 which needed to include the docs/metal_device_create.md changes in the go file (otherwise make generate-docs would override the change, which is what happened here).